torchaudio/prototype/rnnt_pipeline.py

```diff
@@ -0,0 +1,149 @@
+from typing import Callable, List, Tuple
+from dataclasses import dataclass
+import json
+import math
+import os
+import sys
+import torch
+
+import sentencepiece as spm
+import torchaudio
+from torchaudio._internal import download_url_to_file, load_state_dict_from_url
+from torchaudio.prototype import RNNT, RNNTBeamSearch, emformer_rnnt_base
+
+
+__all__ = []
+
+
+_BASE_MODELS_URL = "https://download.pytorch.org/torchaudio/models"
+_BASE_PIPELINES_URL = "https://download.pytorch.org/torchaudio/pipeline-assets"
+
+
+def _download_asset(asset_path: str):
+    dst_path = f"{os.getcwd()}/_assets/{asset_path}"
+    if not os.path.exists(dst_path):
+        os.makedirs(f"{os.getcwd()}/_assets", exist_ok=True)
+        download_url_to_file(f"{_BASE_PIPELINES_URL}/{asset_path}", dst_path)
+    else:
+        sys.stderr.write(f"{asset_path} found at {dst_path}; skipping download.\n")
```

Comment:
The other pipelines do not print anything when the file is already present.
They print message only when download is necessary, so I think we do not need to print that.